easybuild/tools/filetools.py

```diff
@@ -1186,7 +1186,7 @@ def guess_patch_level(patched_files, parent_dir):
     return patch_level
 
 
-def apply_patch(patch_file, dest, fn=None, copy=False, level=None, use_git_am=False):
+def apply_patch(patch_file, dest, fn=None, copy=False, level=None, use_git=False):
```

Comment:
@Flamefire We shouldn't drop `use_git_am` like this, but deprecate it to avoid breaking backwards compatibility:

```python
if use_git_am:
    _log.deprecated('5.0', "Setting 'use_git_am' named argument for apply_patch is deprecated, please set 'use_git=True' instead")
    use_git = True
```

Reply:
https://github.com/easybuilders/easybuild-framework/pull/3414/commits/e5cd2ce2c384285b98b5d1a3bc59f3f91b6f3faf